graylog2-server/src/main/java/org/graylog2/indexer/searches/Searches.java

```diff
@@ -642,6 +641,11 @@ public HistogramResult fieldHistogram(String query,
                 .filter(Objects::nonNull)
                 .collect(Collectors.toList());
 
+            final Optional<String> nonNumericFieldError = errors.stream().filter(error -> error.startsWith("Expected numeric type on field")).findAny();
+            if (nonNumericFieldError.isPresent()) {
```

Comment:
I also would like to move this potentially into `JestUtils.specificException` to have a generic error handling, but this is happening after the release.